src/NLogTarget/NLogTarget.csproj

```diff
@@ -29,6 +29,10 @@
     <PackageReference Include="Desktop.Analyzers" Version="1.1.0">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
```

Comment:
this is default as I understand. Why you need it explicitly?

Reply:
This was inserted by default. I haven't tested without.

Reply:
I'd suggest to remove to avoid confusion